cognite/client/_api/files.py

```diff
@@ -596,6 +597,146 @@ def upload_bytes(
 
         return FileMetadata._load(returned_file_metadata)
 
+    def begin_multipart_upload(
```

Comment:
I would suggest this name
```suggestion
    def multipart_upload_session(
```

Reply:
This method needs to be added to the docs aswell. see `docs/source/files.rst`.